File: task-03/Go/Sub-task 2/sub-task-2.go
package main
import(
    "fmt"
    "io/ioutil"

)

func main() {
    input, err := ioutil.ReadFile("input.txt")
    if err != nil {
        fmt.Println("Failed to read input file: %s", err)
    }


    err = ioutil.WriteFile("output.txt", input, 0644)
    if err != nil {
        fmt.Println("Failed to write to output file: %s", err)
    }
}

File: task-03/Rust/Sub-task 2/sub-task-2.rs
use std::fs::File;
use std::io::{Read, Write};

fn main() {
    
    let mut input_file = File::open("input.txt").expect("Failed to open input file");
    let mut content = String::new();
    input_file.read_to_string(&mut content).expect("Failed to read input file");

    let mut output_file = File::create("output.txt").expect("Failed to create output file");
    output_file.write_all(content.as_bytes()).expect("Failed to write to output file");
}
